micropub.php

```diff
@@ -70,14 +87,30 @@ public static function parse_query($wp) {
       return;
     }
     header('Content-Type: text/plain; charset=' . get_option('blog_charset'));
-
-    $user_id = Micropub::authorize();
+    // For debug purposes be able to bypass Micropub auth with WordPress auth
+    if (MICROPUB_LOCAL_AUTH!=0) {
+      if (!is_user_logged_in()) {
+        auth_redirect();
+      }
+      $user_id = wp_get_current_user();
+    }   
+    else {
+      $user_id = Micropub::authorize();
+    }
+    $headers = getallheaders();
+    if (isset($headers['Accept']) && $headers['Accept']=='application/json') {
+      $json = true;
```

Comment:
style nit: just do `$json = isset($headers['Accept']) && ...` instead of this whole conditional block.